mesh/client.go

```diff
@@ -41,7 +41,7 @@ type MeshClient struct {
 	client meshv1.MeshServiceClient
 }
 
-//bool NameUnique
+// bool NameUnique
```

Comment:
I couldn't find any reason for this comment, could you please explain this? If not needed, document the join function while you're at it ;)